### ISSUE TITLE
Spell check not working.

### ISSUE THREAD

**Issue by halfmanli** [iamhyc/Overleaf-Workshop#206]
### Describe the bug
With incocrret words as input, the VSCode editor does not perform spell check. 
![image](https://github.com/user-attachments/assets/0bc5b1ac-f0b9-40a9-89a4-6660e4499f10)
Usually,  blue wave underlines as such below image are shown, but they suddenly disappear hours ago.
![image](https://github.com/user-attachments/assets/19b41adb-3bf5-4303-a0cb-684de39bcfd4)

### Expected behavior
At the Overleaf web editor, we can see error tips.
![image](https://github.com/user-attachments/assets/85777db8-c442-4b47-86a5-d3e696ecff88)
 
### Environment
 - **Overleaf Workshop Extension version**: [0.14.1]
 - **VS Code version**: [e.g. 1.94.1]
 - **Overleaf Edition**: Official

**Comment by iamhyc** [comment]
Actually, I was working on an Overleaf project now, with the spell check on. It does work, though not perfectly.

Please firstly check if you enable the spell check on the status bar.

![图片](https://github.com/user-attachments/assets/d96f48ad-2033-4a23-933a-38435f7f9b06)

**Comment by halfmanli** [comment]
> Actually, I was working on an Overleaf project now, with the spell check on. It does work, though not perfectly.
> 
> Please firstly check if you enable the spell check on the status bar.
> 
> ![图片](https://private-user-images.githubusercontent.com/9068301/384212373-d96f48ad-2033-4a23-933a-38435f7f9b06.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEwNDQ3OTYsIm5iZiI6MTczMTA0NDQ5NiwicGF0aCI6Ii85MDY4MzAxLzM4NDIxMjM3My1kOTZmNDhhZC0yMDMzLTRhMjMtOTMzYS0zODQzNWY3ZjliMDYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDhUMDU0MTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjQyMTlkNDcyMzhhODg0OTMyY2QzNzJiZThiNDBiZDQzYWI2NTNjNjE3OWFkOWE3NGYzNDQ1YWE3MDhiZDE0MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.CYZdfufj3nG6QblFpuBbxP4u0niQwhgzJ7enzS3aOJE)

Of course, I've checked this multiple times by turning off and on. Here's my status bar:
![image](https://github.com/user-attachments/assets/a5814e6e-92a0-48de-8c01-902a2971c21b)

**Comment by iamhyc** [comment]
> > Actually, I was working on an Overleaf project now, with the spell check on. It does work, though not perfectly.
> > Please firstly check if you enable the spell check on the status bar.

> 
> Of course, I've checked this multiple times by turning off and on. Here's my status bar:

Ok, then please try to attach the developer logs and see if there is any error/warning messages.

> Please find the logs via: Title bar **"Help"** > **"Toggle Developer Tools"** > **"Console"**.

**Comment by halfmanli** [comment]
Here are my operations:
1. In my tex file, I input a random word, and the console shows:
![image](https://github.com/user-attachments/assets/e9e99828-e9cc-408c-b8ea-1c12489f17c8)

**Comment by iamhyc** [comment]
> Here are my operations:
> 
>     1. In my tex file, I input a random word, and the console shows:

I see. So the problem is trying to read a non-existing diagnostic. I will try to reproduce your situation later today.

If possible, please also provide a minimum tex file example to reproduce the situation. Thank you!

**Comment by halfmanli** [comment]
It seems every project is influenced by this bug. For example, I build a new project with the template from: https://www.overleaf.com/latex/templates/ieee-conference-template/grfzhhncsfqn .  The same error arises:
![image](https://github.com/user-attachments/assets/965efec0-493d-40aa-9d58-f3b17f8c1ee6)
 I really appreciate for your help!

**Comment by iamhyc** [comment]
![image](https://github.com/user-attachments/assets/7f8b9b0b-a487-415a-bbbf-b88daee11a86)

It turns out the Official Overleaf Website has offloaded the spell check function to the front-end (i.e., the web browser now do spell check alone without the help from the server), and does not provide spell check function any more.

Please try to use local replica following the guidance [here](https://github.com/iamhyc/Overleaf-Workshop/blob/master/docs/wiki.md#open-project-locally), and refer to local spell check extensions. We have no plan to provide another spell check module.

**Comment by hzphzp** [comment]
Is there any other way to do spell checking in this online overleaf workspace?
